test/e2e/helm/integrations/builder.go

```diff
@@ -0,0 +1,297 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License 2.0;
+// you may not use this file except in compliance with the Elastic License 2.0.
+
+//go:build chart_integrations || e2e
```

Comment:
```suggestion
//go:build helm || e2e
```
should this just be a tag to target the helm chart tests in general?

Reply:
yep makes total sense